### ISSUE TITLE
Multi-line UiCollapsible header

### ISSUE THREAD

**Issue by theshem** [JosephusPaye/Keen-UI#130]
Hi, 

First of all thank you for this great work. I was tweaking the UI components that I encountered [this line](https://github.com/JosephusPaye/Keen-UI/blob/v0.8.9/src/UiCollapsible.vue#L209-L210) within the `UiCollapsible.vue`:

``` sass
.ui-collapsible-header-content {
    @extends $truncate-text;
    line-height: 1.25em;
}
```

And:

``` sass
.ui-collapsible-header {
    // ...
    display: flex;
    height: 0; //  <-- This declaration forces the UA to set the height of the header
               //      to the whatever `min-height` value is set.
   // ...
}
```

What is the reason behind having a single line header hardcoded in the Collapsible component? If one wants to make it stay inline, s/he can use the header `slot` to add the needed HTML having proper styles applied.

---

If you are accepting PR, I can send one :)

**Comment by JosephusPaye** [comment]
We do this because we want to be able to truncate the header and add ellipsis (...) when the text overflows.

If we can retain this behavior and add an option for multiline headers, feel free to send in a pull. However, it'll only be merged for Keen UI v1.0, which is going to be based on Vue 2.

**Comment by JosephusPaye** [comment]
This has been fixed in v1.0.
